app/javascript/shared/serviceWorker.js

```diff
@@ -0,0 +1,17 @@
+window.addEventListener('load', () => {
+  navigator.serviceWorker.register('/service-worker.js').then(registration => {
```

Comment:
We should probably rename this file. This isn't the `serviceWorker`; that's clearly another file. Maybe this could be `serviceWorkerRegisterer.js`? 🤔